### ISSUE TITLE
Bug: XAML Controls Gallery: Accessibility: [MAS 4.2.1]: The name property is not defined for most of the edit field available in the XAML Controls Gallery App. Thus narrator does not read the name of the edit field.

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#4278]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.2.1, Narrator, Sev3, XAML Controls Gallery (Debug), Win32, Wipro

**Steps to reproduce the bug**
1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Status and Info-> Automation Property.
3. Navigate to the "Font Size of the Text" edit box.
4.  Run the Accessibility Insight for the Windows 
5. Observe the name property for the edit fields available in the XAML Controls Gallery App.
6. Also observe the narrator announcement for all the edit field available throughout this app.

**Actual behavior**
The name property is not defined for most of the edit field available in the XAML Controls Gallery App. Narrator says as "edit, 14" for the Font size of text edit field which is having value 14
Note: This issue observed for almost all the edit field available throughout this "XAML Controls Gallery (Debug)" app 

**Expected behavior**
The narrator should read the name of the edit field available on this "XAML Controls Gallery" app 

**User Impact**
If name of the edit field is defined, then it would be very difficulty for the narrator user to understand its purpose exactly.

**Attachment**
[Narrator-EditField-NameProperty.zip](https://github.com/microsoft/microsoft-ui-xaml/files/6020504/Narrator-EditField-NameProperty.zip)

![A11yAuto-ScreenShot](https://user-images.githubusercontent.com/74585893/108682139-f3cd1980-7515-11eb-9a1d-26f193cab626.png)

**Version Info**
Microsoft OS: 21313.1000
App: XAML Controls Gallery (Debug)
